dashboard/src/libs/useModal.tsx

```diff
@@ -208,39 +24,46 @@ const useModal = (options?: {
     return (
       <Dialog.Root open={isOpen()}>
         <Dialog.Portal>
-          <Dialog.Overlay class={overlayStyle} />
-          <DialogPositioner>
+          <Dialog.Overlay
+            class={clsx(
+              'fixed inset-0 bg-black-alpha-600 opacity-0 transition-opacity duration-200',
+              'data-[expanded]:opacity-1',
+            )}
+          />
+          <div class="fixed inset-0 grid place-items-center p-8">
             <Dialog.Content
               onEscapeKeyDown={close}
               onPointerDownOutside={mergedProps.closeOnClickOutside ? close : undefined}
-              as={(
-                asProps: PolymorphicCallbackProps<
-                  ComponentProps<typeof Content>,
-                  DialogContentOptions,
-                  DialogContentRenderProps
-                >,
-              ) => (
-                <Content fit={props.fit} {...asProps}>
-                  {props.children}
-                </Content>
+              as="div"
+              class={clsx(
+                'relative flex max-h-full w-full max-w-142 scale-95 flex-col overflow-hidden rounded-xl bg-ui-primary opacity-1 transition-all duration-300 data-[expanded]:scale-100 data-[expanded]:opacity-100',
```

Comment:
```suggestion
                'relative flex max-h-full w-full max-w-142 scale-95 flex-col overflow-hidden rounded-xl bg-ui-primary opacity-0 transition-all duration-300 data-[expanded]:scale-100 data-[expanded]:opacity-100',
```

多分opacity 0 -> 100 のtransitionが正しい?